agent/check_handler.go

```diff
@@ -31,6 +31,11 @@ const (
 	measureNullStatus = "null-status"
 )
 
+var (
+	dupCheckRequestErr = errors.New("check request has already been received - agent & check may have multiple matching subscriptions")
```

Comment:
At least for me `&` doesn't get decoded and ends up looking like this:
```
check request has already been received - agent \u0026 check may...
```

Mind using "and" instead?

Reply:
Nice catch, fixed!